wai-aria/role/tab-roles.html

```diff
@@ -0,0 +1,30 @@
+<!doctype html>
+<html>
+<head>
+    <title>Tab-related Role Verification Tests</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="/resources/testdriver.js"></script>
+    <script src="/resources/testdriver-vendor.js"></script>
+    <script src="/resources/testdriver-actions.js"></script>
+    <script src="/wai-aria/scripts/aria-utils.js"></script>
+</head>
+<body>
+
+<p>Tests <a href="https://w3c.github.io/aria/#tab">tab</a> and related roles.</p>
+
+<div role="tablist" data-testname="div role is tablist" data-expectedrole="tablist" class="ex">
+    <div role="tab" data-testname="role is tab (in div tablist)" data-expectedrole="tab" class="ex">x</div>
```

Comment:
I think this should include some selected and unselected tabs too... I don't anticipate that will change the result, but good to have a test that confirms it.

Reply:
likewise a focusable versus non-focusable... maybe a static "roving" tab index... No need to hook up the JS, but something like `aria-selected="true" tabindex="0"` on the selected tab, and `tabindex="-1"` on the unselected tabs.

Reply:
Added examples with selected/unselect tabs, and several roving tabindex examples.